justfile

```diff
@@ -122,9 +122,15 @@ _generate_bridge_pure_dart_multi:
 _generate_bridge_with_flutter:
     {{cargo_run_codegen}} \
         --rust-input frb_example/with_flutter/rust/src/api.rs \
-        --dart-output frb_example/with_flutter/lib/bridge_generated.dart \
-        --c-output frb_example/with_flutter/ios/Runner/bridge_generated.h \
+        --rust-input frb_example/with_flutter/rust/src/api2.rs \
```

Comment:
nit: So, shall we use `pure_dart_multi` as a test bed for this PR?

Reply:
"We can use the `pure_dart_multi` example. However, since `dart-decl-output` is not set in this example, we won't be able to see the changes in this PR and won't be able to catch any future errors. Maybe we should add a fourth example called `pure_dart_multi_with_decl_output`. 

However, I still prefer having more examples on the Flutter side. So we can add the fourth example on the Flutter side, like `with_flutter_multi`."

Reply:
As for more testing examples, possibly related to: https://github.com/fzyzcjy/flutter_rust_bridge/issues/1210